public/js/pimcore/element/helpers/gridColumnConfig.js

```diff
@@ -544,8 +544,8 @@ pimcore.element.helpers.gridColumnConfig = {
             var editor = new pimcore.object.tags[tagType](null, fieldInfo.layout.layout);
             editor.setObject(this.object);
         } else {
-            var tagType = this.fieldObject[fieldInfo.dataIndex].layout.fieldtype;
-            let layoutInfo = this.fieldObject[fieldInfo.dataIndex].layout
+            let layoutInfo = this.fieldObject[fieldInfo.dataIndex].layout;
+            const tagType = this.fieldObject[fieldInfo.dataIndex].type ?? layout.fieldtype;
```

Comment:
The console log is
![image](https://github.com/pimcore/admin-ui-classic-bundle/assets/6014195/eb726a45-04db-4e1c-b8bb-57c6eab6ad51)
Don't know when and where the `fieldtype` was used, maybe a typo but kept as fallback

```suggestion
            const tagType = this.fieldObject[fieldInfo.dataIndex].type;
```
in case, could just removed